src/rosconsole/impl/rosconsole_log4cxx.cpp

```diff
@@ -84,27 +84,27 @@ class ROSConsoleStdioAppender : public log4cxx::AppenderSkeleton
   }
 
 protected:
-  virtual void append(const log4cxx::spi::LoggingEventPtr& event, 
+  virtual void append(const log4cxx::spi::LoggingEventPtr& event,
```

Comment:
Please revert unrelated whitespace changes like this.

Reply:
reverted.